ENIGMAsystem/SHELL/Platforms/xlib/XLIBwindow.cpp

```diff
@@ -517,13 +517,25 @@ void window_set_rectangle(int x, int y, int w, int h) {
 // FULLSCREEN //
 ////////////////
 
+static bool prefer_sizeable = false;
+static bool initial_fullscreen = true;
```

Comment:
Please move these into the `window_set_fullscreen` method to avoid the temptation for other routines to misuse them.

Reply:
I would do that actually if it wouldn't break perfectly good working code, but it would....

Reply:
I misspoke; what you said seems correct.

Reply:
I tried exactly what you said and it broke the ability to set a sizeable window via global game settings. Reverted.

Reply:
You dropped the word "static" when you moved them.

Reply:
Interesting, I though the static keyword would make the value constant like const does if put in a function. I'm not sure why I thought that... Anyway, that did the trick, thanks!